airflow/providers/google/cloud/example_dags/example_dataproc.py

```diff
@@ -252,3 +263,40 @@
 
     # Task dependency created via `XComArgs`:
     #   spark_task_async >> spark_task_async_sensor
+
+with models.DAG(
+    "example_gcp_batch_dataproc",
+    schedule_interval='@once',
+    start_date=days_ago(1),
```

Comment:
```suggestion
    "example_gcp_batch_dataproc",
    schedule_interval='@once',
    start_date=datetime(2021, 1, 1),
    catchup=False,
```

There is an almost-finished effort to transition away from using `start_date=days_ago(n)` in example DAGs to using a static value as best practice. New example DAGs should follow the static start_date approach. The value used doesn't matter as long as it's static.

Also adding `catchup=False` has been recently discussed for all example DAGs to help with any accidental DAG run explosions if new users mutate the `schedule_interval` without fully understanding `catchup=True` is the default.

Reply:
@josh-fell I've changed this code to the new approach